.github/workflows/pull_request.yaml

```diff
@@ -12,9 +12,12 @@ jobs:
     with:
       owner: ${{ github.repository_owner }}
       trivy-image-config: "trivy.yaml"
-      multiarch-awareness: true
+      multiarch-awareness: false
       cache-action: ${{ (github.event_name == 'push') && 'save' || 'restore' }}
-      platform-labels: '{"arm64": ["Ubuntu_ARM64_4C_16G_01"]}'
+      # pinning to use rockcraft 1.3.0 feature `entrypoint-service`
+      rockcraft-revision: "1783"
```

Comment:
let's remember to update this once rockcraft 1.3.0 is out

Reply:
Agree, it will need to be unpinned at some point that rockcraft 1.3.0 is latest/stable